main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
Ok, then please rename it to `main.example.go` or some such, and add a comment at the top of the file indicating the intention.